javax.script.ScriptException: java.lang.IllegalStateException: Unknown license set identifier: permissive.

I have been following same procedure mentioned in [Getting-started](https://github.com/heremaps/oss-review-toolkit/blob/master/docs/getting-started.md#getting-started) guide.

While Running the evaluator [evaluator](https://github.com/heremaps/oss-review-toolkit/blob/master/docs/getting-started.md#6-running-the-evaluator) command

It threw IllegalStateException: Unknown license set identifier: permissive

### 

> dinesh@dinesh-VirtualBox:~/ort/oss-review-toolkit/cli/build/install/ort/bin$ ./ort evaluate --rules-file /home/dinesh/ort/oss-review-toolkit/docs/examples/rules.kts -i /home/dinesh/ort/output/scan-result.yml -o /home/dinesh/ort/output/mime-types
> ________ _____________________
> \_____  \\______   \__    ___/ the OSS Review Toolkit, version 0.1.0-SNAPSHOT.
>  /   |   \|       _/ |    |    Running 'evaluate' under Java 1.8.0_242 on Linux with
> /    |    \    |   \ |    |    ORT_USER_HOME = /home/dinesh/.ort
> \_______  /____|_  / |____|    SHELL = /bin/bash
>         \/       \/
> More environment variables:
> TERM = xterm-256color
> 
> Exception in thread "main" javax.script.ScriptException: java.lang.IllegalStateException: Unknown license set identifier: permissive.
> 	at org.jetbrains.kotlin.cli.common.repl.KotlinJsr223JvmScriptEngineBase.asJsr223EvalResult(KotlinJsr223JvmScriptEngineBase.kt:101)
> 	at org.jetbrains.kotlin.cli.common.repl.KotlinJsr223JvmScriptEngineBase.compileAndEval(KotlinJsr223JvmScriptEngineBase.kt:63)
> 	at kotlin.script.experimental.jvmhost.jsr223.KotlinJsr223ScriptEngineImpl.compileAndEval(KotlinJsr223ScriptEngineImpl.kt:93)
> 	at org.jetbrains.kotlin.cli.common.repl.KotlinJsr223JvmScriptEngineBase.eval(KotlinJsr223JvmScriptEngineBase.kt:31)
> 	at javax.script.AbstractScriptEngine.eval(AbstractScriptEngine.java:264)
> 	at com.here.ort.utils.ScriptRunner.run(ScriptRunner.kt:68)
> 	at com.here.ort.evaluator.Evaluator.run(Evaluator.kt:63)
> 	at com.here.ort.commands.EvaluatorCommand$run$evaluatorRun$2.invoke(EvaluatorCommand.kt:149)
> 	at com.here.ort.commands.EvaluatorCommand$run$evaluatorRun$2.invoke(EvaluatorCommand.kt:55)
> 	at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
> 	at com.here.ort.commands.EvaluatorCommand.run(EvaluatorCommand.kt:152)
> 	at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:139)
> 	at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:14)
> 	at com.github.ajalt.clikt.core.CliktCommand.parse(CliktCommand.kt:215)
> 	at com.github.ajalt.clikt.core.CliktCommand.parse$default(CliktCommand.kt:212)
> 	at com.github.ajalt.clikt.core.CliktCommand.main(CliktCommand.kt:230)
> 	at com.github.ajalt.clikt.core.CliktCommand.main(CliktCommand.kt:253)
> 	at com.here.ort.OrtMainKt.main(OrtMain.kt:205)
> Caused by: java.lang.IllegalStateException: Unknown license set identifier: permissive.
> 	at com.here.ort.model.licenses.LicenseConfiguration.getLicensesForSet(LicenseConfiguration.kt:76)
> 	at ScriptingHost51b292f1_Line_1.getLicenseSet(ScriptingHost51b292f1_Line_1.kts:43)
> 	at ScriptingHost51b292f1_Line_1.<init>(ScriptingHost51b292f1_Line_1.kts:45)
> 	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
> 	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
> 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
> 	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
> 	at kotlin.script.experimental.jvm.BasicJvmScriptEvaluator.evalWithConfigAndOtherScriptsResults(BasicJvmScriptEvaluator.kt:95)
> 	at kotlin.script.experimental.jvm.BasicJvmScriptEvaluator.invoke$suspendImpl(BasicJvmScriptEvaluator.kt:40)
> 	at kotlin.script.experimental.jvm.BasicJvmScriptEvaluator.invoke(BasicJvmScriptEvaluator.kt)
> 	at kotlin.script.experimental.jvmhost.repl.JvmReplEvaluator$eval$$inlined$write$lambda$2.invokeSuspend(legacyReplEvaluation.kt:57)
> 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
> 	at org.jetbrains.kotlin.kotlinx.coroutines.DispatchedTask.run(Dispatched.kt:238)
> 	at org.jetbrains.kotlin.kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.kt:116)
> 	at org.jetbrains.kotlin.kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:80)
> 	at org.jetbrains.kotlin.kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:54)
> 	at org.jetbrains.kotlin.kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
> 	at org.jetbrains.kotlin.kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:36)
> 	at org.jetbrains.kotlin.kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
> 	at kotlin.script.experimental.jvmhost.repl.JvmReplEvaluator.eval(legacyReplEvaluation.kt:57)
> 	at org.jetbrains.kotlin.cli.common.repl.GenericReplCompilingEvaluatorBase.eval(GenericReplCompilingEvaluator.kt:76)
> 	at org.jetbrains.kotlin.cli.common.repl.GenericReplCompilingEvaluatorBase.compileAndEval(GenericReplCompilingEvaluator.kt:51)
> 	at org.jetbrains.kotlin.cli.common.repl.KotlinJsr223JvmScriptEngineBase$compileAndEval$1.invoke(KotlinJsr223JvmScriptEngineBase.kt:64)
> 	at org.jetbrains.kotlin.cli.common.repl.KotlinJsr223JvmScriptEngineBase$compileAndEval$1.invoke(KotlinJsr223JvmScriptEngineBase.kt:26)
> 	at org.jetbrains.kotlin.cli.common.repl.KotlinJsr223JvmScriptEngineBase.asJsr223EvalResult(KotlinJsr223JvmScriptEngineBase.kt:90)
> 	... 17 more


I am using the [rules.krs](https://github.com/heremaps/oss-review-toolkit/blob/master/docs/examples/rules.kts) as part of evaluationo. It threw above error. Any help?

--- a/docs/getting-started.md
+++ b/docs/getting-started.md
@@ -221,6 +221,19 @@ To scan the source code of `mime-types` and its dependencies the source code of 
 needs to be downloaded. The _downloader_ tool could be used for this, but it is also integrated in the `scanner` tool,
 so the scanner will automatically download the source code if the required VCS metadata could be obtained.
 
+Note that if _downloader_ is unable to download the source code due to say a missing
+source code location in the package metadata then you can use curations as a workaround.
+
+To use curations, create a [curations.yml](config-file-curations-yml.md)
+and pass it to the `--package-curations-file` option of the _analyzer_:
+
+```
+cli/build/install/ort/bin/ort analyze
+  -i [mime-types-path]
+  -o [analyzer-output-path]
+  --package-curations-file [ort-configuration-path]/curations.yml
+```
+
 ORT is designed to integrate lots of different scanners and is not limited to license scanners, technically any tool
 that explores the source code of a software package could be integrated. The actual scanner does not have to run on the
 same machine, for example we will soon integrate the [ClearlyDefined](https://clearlydefined.io/) scanner backend which
@@ -265,17 +278,24 @@ on a bigger project you will see that `ScanCode` often finds more licenses than 
 
 ## 6. Running the evaluator
 
-The evaluator can apply a set of rules against the scan result created above. ORT provides an
-[example of a policy rules file](examples/rules.kts) that can be used for testing the _evaluator_. 
+The evaluator can apply a set of rules against the scan result created above.
+ORT provides examples for the policy rules file [(rules.kts)](examples/rules.kts),
+[user-defined categorization of licenses (licenses.yml)](examples/licenses.yml) and
+[user-defined package curations (curations.yml)](examples/licenses.yml) that can be used for testing the _evaluator_. 
 
 To run the example rules use:
 
 ```bash
 cli/build/install/ort/bin/ort evaluate
+  --package-curations-file curations.yml
   --rules-file rules.kts
+  --license-configuration-file licenses.yml
   -i [scanner-output-path]/scan-result.yml
   -o [evaluator-output-path]/mime-types
 ```
+
+See the [curations.yml documentation](config-file-curations-yml.md) to learn more about using curations to correct invalid or missing package metadata
+and the [licenses.yml documentation](config-file-licenses-yml.md) on how you can classify licenses to simplify writing the policy rules.
 
 It is possible to write your own evaluator rules as a Kotlin script and pass it to the _evaluator_ using `--rules-file`.
 Note that detailed documentation for writing custom rules is not yet available.

